CONTRIBUTING.md

```diff
@@ -45,25 +47,23 @@ Some of the best content contributions come during the PR review cycles. To ensu
 
 ## Pull Requests (PRs)
 
-Provide a helpful, informative PR title that include the context and page you are updating. Each PR must address one issue or feature.
+To provide a fix or update, make a direct contribution. We appreciate your contributions!
```

Comment:
Stylisitic nitpick: the repitition of 'contribution[s]' breaks flow a little.
Here's a suggestion but of course feel free to improve!
> Feel free to submit a fix or update directly. We appreciate your contributions!

Reply:
oops yes of course